### ISSUE TITLE
Is this plug-and-play compatible with the current Perl6::Documentable?

### ISSUE THREAD

**Issue by JJ** [antoniogamiz/Perl6-Documentable#53]
If I drop the current Perl6::Documentable in the doc repo and and this one, will it work? Have you tested it?

**Comment by antoniogamiz** [comment]
Yes, it is. At this moment, if you setup the dependencies described [here](https://github.com/antoniogamiz/Perl6-Documentable#dependencies-installation), you only need to execute `documentable setup` and `documentable start -a --highlight` to generate a fully functional documentation.
